### PR TITLE
Add parsing of lua files in aircraft root folder and in Custom Avioni…

### PR DESCRIPTION
**Pull request by sparker256**
…cs folder

### CODE REVIEW

**Comment by leecbaker** [comment]
Looks good- it will be a couple days until I can verify that it works, then well merge it. Thanks!

File: src/find_datarefs_in_files.cpp [hunk 1]
```diff
@@ -136,7 +136,30 @@ std::vector<std::string> getDatarefsFromAircraft(const std::string & acf_path) {
...
+				std::vector<std::string> refs = getDatarefsFromFile(path.string());
+				all_refs.insert(all_refs.begin(), refs.begin(), refs.end());
+			}
+		}
+	}
+
+	//lua files. Recurse over Custom Avionics directory structure
+	paths.push_back(aircraft_dir / "Custom Avionics");
+
+	while(false == paths.empty()) {
```
Comment (leecbaker) [inline src/find_datarefs_in_files.cpp]: Bill this loop duplicates some of the functionality of the while() loop in the code below. I've tried to basically copy this functionality into 636dfbd0cc292e568b7cc83b45b053011e04c7ff, can you verify that commit finds the datarefs that you were missing?

**Comment by leecbaker** [comment]
Thanks for sending this pull request- looks like we can make life easier for some people using Lua in their aircraft!

File: src/find_datarefs_in_files.cpp (hunk 1 above)
Comment (sparker256) [inline src/find_datarefs_in_files.cpp]: Checked it on a couple of aircraft and it is finding them corectly.  Thanks

**Pull request closed without merging**